pkg/controller/ingressconfig/ingressconfig_controller.go

```diff
@@ -64,7 +63,8 @@ var _ reconcile.Reconciler = &ReconcileIngressConfigs{}
 // ReconcileIngressConfigs watches for updates to ingress controller configuration
 // and sets the network policy related labels on the openshift-host-network namespace
 type ReconcileIngressConfigs struct {
-	client crclient.Client
+	client cnoclient.Client
+	mgr    manager.Manager
```

Comment:
Instead of adding the manager to the controller you can extract the client from `cnoclient.Client` with `client.Default().CRCClient()`